packages/vscode-extension/src/commonlib/codeFlowLogin.ts

```diff
@@ -332,14 +332,6 @@ export class CodeFlowLogin {
             }
           })
           .catch(async (error) => {
-            VsCodeLogInstance.error(
```

Comment:
Instead of totally removing, is it possible to log as debug or verbose? At least we can debug ourselves...

Reply:
Anyway, ok to remove as well.